<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SleepScreen</class>
 <widget class="QWidget" name="SleepScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="wakeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>900</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(0, 0, 0);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>enter PIN then tap anywhere to wake (default pin is 1234)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pin1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="pin2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="pin3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="pin4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
